specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4143,6 +4181,24 @@
       "type": "string",
       "in": "path",
       "x-ms-skip-url-encoding": true
+    },
+    "RecognitionModel": {
+      "name": "RecognitionModel",
+      "description": "Recognition model name. Recognition model is used when the face features are extracted, so a recognition model version could be provided when performing a Detection. If the user does not provide recognition model, default is 'recognition_v01'.",
+      "required": false,
+      "type": "string",
+      "in": "query",
+      "x-ms-parameter-location": "method",   
+      "default": "recognition_v01",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "RecognitionModel",
+        "modelAsString": false
```

Comment:
Is there any scenario where an older client may see a newer value for recognition model? For example, client using a newer version of the SDK would use the (at some point in the future introduced recognition_v03) model and the older client could list and see recognized entities that had this new value? 

If so, the modelAsString: false will cause problems.

Reply:
Thanks @johanste to point out this! I have never really understood this `modelAsString` parameter before. Yes, there may be a chance we introduce new recognition model (e.g. _v03) in the future and the API will return the new model if exists in many interfaces e.g. PersonGroup - List.

We will change it to `true` in this PR for swagger, while this should not block the API review I think.

Reply:
@acured Please update this and also check whether other places need similar changes. Let me know if you do not understand it, thanks.

Reply:
Done.